src/opendr/perception/object_detection_2d/nanodet/algorithm/nanodet/model/backbone/ghostnet.py

```diff
@@ -39,7 +38,7 @@ def _make_divisible(v, divisor, min_value=None):
     new_v = max(min_value, int(v + divisor / 2) // divisor * divisor)
     # Make sure that round down does not go down by more than 10%.
     if new_v < 0.9 * v:
-        new_v += divisor
+        new_v = new_v + divisor
```

Comment:
```suggestion
        new_v += divisor
```

Reply:
In Onnx with pytorch documentation it has a comment to Avoid in-place operations in some scenarios. I am not sure if it is ok in this particular senario so I change it.